### PR TITLE
WIP: added flask cors instead of restplus one

### PR DESCRIPTION
**Pull request by quaxsze**
fix https://github.com/opendatateam/udata/issues/2481

### CODE REVIEW

File: udata/api/__init__.py [hunk 1]
```diff
@@ -61,6 +62,8 @@
     'X-Forwarded-Proto',
 )
 
+co = CORS(allow_headers=PREFLIGHT_HEADERS)
```
Comment (abulte) [inline udata/api/__init__.py:65]: avoid two letters variable names. `cors`?

File: udata/api/__init__.py [hunk 2]
```diff
@@ -165,12 +168,7 @@ def page_parser(self):
 
 api = UDataApi(
     apiv1,
-    decorators=[csrf.exempt,
-                cors.crossdomain(origin='*',
-                                 credentials=True,
```
Comment (quaxsze) [inline udata/api/__init__.py:170]: Credentials are set to true here.
Flask-cors has an option 'supports_credentials' but as the documentation states 'This option cannot be used in conjuction with a ‘*’ origin'.
Do we need to refine the origin parameter in order to use this option?

Comment (abulte) [reply udata/api/__init__.py:170]: good question, let's see what works and doesn't work on demo as is.

Comment (abulte) [reply udata/api/__init__.py:170]: (I don't see how we can refine the origin 🤔)

**Comment by quaxsze** [comment]
Handled in https://github.com/opendatateam/udata/pull/2510

**Pull request closed without merging**